src/examples/chrome/SheetSize.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Sheet } from '@zendeskgarden/react-chrome';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Fieldset, Field, Radio, Label } from '@zendeskgarden/react-forms';
+import { getColor } from '@zendeskgarden/react-theming';
+
+const StyledFieldset = styled(Fieldset)`
+  margin: ${props => props.theme.space.md};
+`;
+
+const StyledRow = styled(Row)`
+  border: ${props => props.theme.borderWidths.sm} dashed;
+  border-color: ${props => getColor('neutralHue', 400, props.theme)};
+  overflow: auto;
+`;
+
+const StyledSheetFooterItem = styled(Sheet.FooterItem)`
+  &:first-child {
+    margin-left: 0;
+  }
+`;
+
+const [col4, col6, px380, px480] = [
+  { label: '4 columns', sheet: '100%', cols: 4 },
+  { label: '6 columns', sheet: '100%', cols: 6 },
+  { label: '380 pixels', sheet: '380px', cols: 'auto' },
+  { label: '480 pixels', sheet: '480px', cols: 'auto' }
+];
+
+const Example = () => {
+  const [size, setSheetSize] =
+    useState<{ label: string; sheet: string; cols: string | number }>(px380);
+
+  return (
+    <Grid gutters={false}>
+      <Row>
+        <Col>
+          <StyledFieldset>
+            <StyledFieldset.Legend>Sheet size</StyledFieldset.Legend>
+            {[col4, col6, px380, px480].map(sampleSize => (
+              <Field key={`sizes-${sampleSize.label.replace(' ', '-')}`}>
+                <Radio checked={size === sampleSize} onChange={() => setSheetSize(sampleSize)}>
+                  <Label>{sampleSize.label}</Label>
+                </Radio>
+              </Field>
+            ))}
+          </StyledFieldset>
+        </Col>
+      </Row>
+
+      <StyledRow justifyContent="end">
+        <Col size={size.cols}>
+          <Sheet isOpen isAnimated={false} size={size.sheet}>
+            <Sheet.Header>
+              <Sheet.Title>Size</Sheet.Title>
+              <Sheet.Description>Sizing the Sheet</Sheet.Description>
```

Comment:
The same update here (as above comment) due to axe raising this as an issue.

Reply:
ditto comment reply.